lib/app/layouts/conversation_view/colors/voice_message_recorder_colors.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/cupertino.dart';
+
+const Color iOSWavesColor = CupertinoDynamicColor.withBrightness(
+  color: Color(0xFFFF3B30),
+  darkColor: Color(0xFFFF1B30),
+);
+
+const Color iOSVoiceRecorderBackgroundColor = CupertinoDynamicColor.withBrightness(
+  color: Color(0xFFFFF5F4),
+  darkColor: Color(0xFF1C0606),
+);
+
+const Color iOSStopIconColor = CupertinoDynamicColor.withBrightness(
+  color: Color(0xFFFF3B30),
+  darkColor: Color(0xFFff6f61),
+);
+
+const Color iOSStopIconBackground = CupertinoDynamicColor.withBrightness(
+  color: Color(0xFFFFD2CF),
+  darkColor: Color(0xFF7E1116),
+);
```

Comment:
While mimicking the iOS colors would be nice, I fear that this would cause a lot of issues due to the vast theming options we have. Like lets say someone's material you theme is purple, this orange color would look pretty bad. As such, I recommend removing these custom colors - I will explain on each code block how you should color them to better fit in.